source/develop/ao2mo_developer.rst

```diff
@@ -1,6 +1,127 @@
 .. _developer_ao2mo:
 
+***********************
 AO to MO transformation
 ***********************
 
 *Modules*: :mod:`ao2mo`
+
+Overview
+========
+The :mod:`ao2mo` module exposes users with :func:`kernel`, :class:`load`, :func:`resture` to handle 4-index AO to MO transformation.
+The function :func:`kernel` takes in :class:`Mole` object or the ao integrals stored in a :class:`numpy.ndarray` and a set of mo coefficients
+If ``erifile`` is specified, the integrals will be stored in a HDF5 file, otherwise it will be stored in a numpy array.
+::
```

Comment:
Hi :)! I haven't read through this yet, but I see that this first example here at least is quite complete (with all the imports etc). Do you think it might make sense to move it into an example and import it?

Reply:
Probably, I was thinking of them as a single runnable script that shows all different usage, but it might be a duplicate of existing examples. Btw, does anyone know how are the examples indexed? The indices of existing examples seems rather random to me.

Reply:
I think there are also very small examples, see e.g. https://github.com/pyscf/pyscf/blob/master/examples/cc/00-simple_ccsd.py . I am not sure how they are indexed to be honest. Probably starting from 00, 01, .... I have always assumed that sometimes examples are used in multiple directories so that is why the numbering scheme makes a jump but I don't actually know.

Reply:
I'll merge this for the upcoming alpha release; can you add the examples in PySCF and `include` them in the docs in a future PR?